src/components/atoms/NetworkIndicator.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react'
+import Tooltip from '@material-ui/core/Tooltip'
+import CheckCircleOutlineOutlinedIcon from '@material-ui/icons/CheckCircleOutlineOutlined'
+import ErrorIcon from '@material-ui/icons/Error'
+import WarningIcon from '@material-ui/icons/Warning'
+import { green, yellow } from '@material-ui/core/colors'
+
+export interface NetworkIndicatorProps {
+  iconClassName?: string
+  currentNetworkId?: number
+  requiredNetworkId?: number
+  onRightNetworkMessage?: string
```

Comment:
Can we rename this to `onCorrectNetwork` or `onMatchingNetwork`?

Reply:
> Do we still need to commit the `dist` folder?

Good point. Only for testing purposes but yes, we can finally get rid off that